gitwork/gitwork.go

```diff
@@ -0,0 +1,159 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"gopkg.in/src-d/go-git.v4"
+	. "gopkg.in/src-d/go-git.v4/_examples"
+	"gopkg.in/src-d/go-git.v4/plumbing"
+)
+
+var repoPath = os.Getenv("GOPATH") + "/src/github.com/litmuschaos/chaos-charts/"
+var tempCommit string
+var masterCommit string
```

Comment:
let's make a struct here, something like
```go
type GitConfig struct {
	RepositoryName string
	Branch         string
	RepositoryURL  string
	LocalCommit    string
	RemoteCommit   string
}
```
**Not exactly like this**, something like this

Reply:
was not really using **branch**, so except that implemented this struct